pkg/kubernetes/diff.go

```diff
@@ -23,7 +23,15 @@ Please upgrade kubectl to at least version 1.18.1.`)
 	// required for separating
 	namespaces, err := k.ctl.Namespaces()
 	if err != nil {
-		return nil, errors.Wrap(err, "listing namespaces")
+		resourceNamespaces := findNamespaces(state)
+		namespaces = map[string]bool{}
+		for namespace := range resourceNamespaces {
+			_, err = k.ctl.Namespace(namespace)
+			if err != nil {
+				return nil, errors.Wrap(err, "retrieving namespaces")
```

Comment:
This misses the case where a namespaces does not exist on the server, which is non-fatal but should just result in the ns not being added to the list, so the following code can handle accordingly

Reply:
Added `--ignore-not-found` so `kubectl` now doesn't report an error on namespace not found. This makes it much easier to catch the condition we are after.